Added ASL tutorial

zoebentley opened the pull request:
Changes: 
- adds thumbnail file for Getting Started ASL to "thumbnails" folder
- adds wistia link to translate-video.JS
- links the thumbnail file in "index.jsx"
- adds the Getting Started ASL tutorial to the tutorials in "index.jsx"
- makes it so that the original Getting Started video points to "Getting Started ASL" as one if its 2 follow-up tutorial suggestions

Commit:
Added ASL tutorial

Commit:
lint

zoebentley committed:
Merge branch 'develop' into getting-started-asl

seotts commented on src/lib/libraries/decks/index.jsx:
```diff
@@ -105,14 +108,36 @@ export default {
         }, {
             deckIds: [
-                'add-a-backdrop',
+                'getting-started-ASL',
```
```suggestion
                'intro-getting-started-ASL',
```

seotts replied:
If I understand how this file works, I think it should be this.

zoebentley replied:
Oh - thanks so much for the catch!

seotts commented on src/lib/libraries/decks/index.jsx:
```diff
@@ -105,14 +108,36 @@ export default {
+        name: (
+            <FormattedMessage
+                defaultMessage="Getting Started - ASL"
+                description="Name for the 'Getting Started ASL' how-to"
```
I wonder if it would be helpful for the translators to have ASL written out as "American Sign Language" in the description text?

zoebentley replied:
Sounds great

seotts reviewed (changes requested):
This is awesome. I found one small (but consequential) bug, and also made one other minor suggestion.

zoebentley committed:
Update src/lib/libraries/decks/index.jsx

Co-authored-by: Sarah Otts <se0042@gmail.com>

zoebentley committed:
Update index.jsx

zoebentley committed:
Update index.jsx

Realized that I need to discuss this change with members of the resources team. The rest of the PR is good to go

Commit:
Merge branch 'develop' into getting-started-asl

Commit:
Merge branch 'develop' into getting-started-asl
